src/css/map-view.css

```diff
@@ -1665,3 +1665,99 @@ other class: .ui-slider-range */
   }
 }
 /** END - Expansion panel View. */
+
+/*****************************************************************************************
+ *
+ * Legend
+ *
+ * The full legend that overlays the map
+ *
+ */
+
+.legend-container {
+  z-index: var(--map-z-index-base);
+  position: absolute;
+  left: 1rem;
+  bottom: 1rem;
+  border-radius: var(--map-border-radius-big);
+  background: var(--portal-col-neutral-00, var(--map-col-bkg__deprecate));
+  width: 6.875rem;
+
+  .legend-container__header {
```

Comment:
I know metacatui is very inconsistent in our CSS conventions, but we're gradually trying to move toward BEM principles. Would it be a big refactor to move these elements around a bit to make rules less specific? Here, for example, `.legend-container__header` does not need to be nested under `.legend-container` - it gives `legend-container__header` a higher specificity than it needs. With SCSS/SASS, I know we would nest it like so:

```
.legend-container {
  &__header {
    // styles
  }
}
```

which is nicer for readability. But since we're just using plain CSS, I think it would be better to move `.legend-container__header` out of `.legend-container` and just make it a sibling. This way, we can avoid the specificity issue. So:

```
.legend-container {
  // styles
}

.legend-container__header {
  // styles
}
```

What do you think?

Reply:
^ If you think it's a good idea, I could make an issue for this and we can address it separately

Reply:
addressed in #2446 . otherwise the conflicts will be 🫠